MessengerProj/src/main/res/values/strings.xml

```diff
@@ -417,14 +417,14 @@
     <string name="Never">Never</string>
     <string name="AutocryptKeyTransferInitiate">Initiate Autocrypt Key Transfer</string>
     <string name="AutocryptKeyTransfer">Autocrypt Key Transfer</string>
-    <string name="AutocryptKeyTransferMsgBeforeSend">The <![CDATA[<i>]]>Autocrypt Key Transfer<![CDATA[</i>]]> requires that the mail client on the other device is Autocrypt-compliant.\n\nYou can then send your secret key to yourself. The key will be encrypted by a setup code which is displayed here and must be typed on the other device.</string>
-    <string name="AutocryptKeyTransferMsgAfterSend">Your key has been sent to yourself. Switch to the other device and open the setup message. You should be prompted for a setup code. Type the following digits into the prompt:\n\n%1$s\n\nOnce you\'re done, your other device will be ready to use Autocrypt.</string>
+    <string name="AutocryptKeyTransferMsgBeforeSend">The <![CDATA[<i>]]>Autocrypt Key Transfer<![CDATA[</i>]]> requires the e-mail client on the other device to be Autocrypt-compliant.\n\nYou can then send your secret key to yourself. It will be encrypted by a setup code, displayed here to be typed onto the other device.</string>
+    <string name="AutocryptKeyTransferMsgAfterSend">Your key has been sent to yourself. Open the setup message on the other device and type the following digits:\n\n%1$s\n\nOnce done, your other device will be ready to use Autocrypt.</string>
     <string name="AutocryptSetupMessageSubject">Autocrypt Setup Message</string>
-    <string name="AutocryptSetupMessageGeneralBody">This is the Autocrypt Setup Message used to transfer your key between clients.\n\nTo decrypt and use your key, open the message in an Autocrypt-compliant client and enter the setup code presented on the generating device.</string>
-    <string name="AutocryptSetupMessageTapBody">This is the Autocrypt Setup Message used to transfer your key between clients.\n\nTo decrypt and use your key, tap on this message.</string>
-    <string name="AutocryptKeyTransferPleaseEnterCode">Please enter the setup code that is displayed on the other device.</string>
-    <string name="AutocryptKeyTransferBadCode">Bad setup code. Please try again.\n\nIf you do not remember the setup code, just initiate the key transfer on the other device again.</string>
-    <string name="AutocryptKeyTransferSucceeded">Key transferred successfully. This device is now ready to use Autocrypt with the same key as the other device.</string>
+    <string name="AutocryptSetupMessageGeneralBody">The setup code must be entered on the other device to use your key there.</string>
+    <string name="AutocryptSetupMessageTapBody">Use this Autocrypt setup message to transfer your key between clients.\n\nTo decrypt and use your key, tap this message.</string>
```

Comment:
This is the part that says "tap", which still states "Autocrypt setup message".
Does Autocrypt compliance somehow require a touchscreen?

Reply:
*GeneralBody is the text that is written to the mail and may be displayed by any client.
*TapBody is the text that is displayed by Delta Chat on receiving a setup message (instead of the original text which is *GeneralBody if the message comes from another Delta client or a completely different text from other clients)

So, inside Delta Chat, the *GeneralBody messages are never displayed (exception: older Delta Chat clients that do not process the Setup message).

See https://delta.chat/assets/blog/20171215-asm-delta-enigmail.jpg to get an idea: the text in thunderbird is not the text provided by Delta Chat ...